src/pydrex/core.py

```diff
@@ -714,6 +714,12 @@ def _get_rotation_and_strain(
     """
     crss = get_crss(phase, fabric)
     slip_invariants = _get_slip_invariants(strain_rate, orientation)
+    # Handle the case where all slip invariants are zero,
+    # thus no slip is possible (occurs for a few specific orientations and strain rates,
+    # for example in the case of uniaxial compression and an identity orientation
+    # i.e. grain orientation aligned to the coordinate system).
+    if min(slip_invariants) == max(slip_invariants) == 0.0:
```

Comment:
`not slip_invariants.any()` or `np.all(slip_invariants == 0)`
What do you think?

Reply:
Sure, changed to the second one.